fudge-ratio/src/main/webapp/calculator/main.js

```diff
@@ -1,83 +1,16 @@
-import { saveSessionState, recoverSessionState } from "../code/save.js";
-import { cleanSessionState, isNotEmpty } from "../code/save.js";
-import loadActivities from "../code/loadActivities.js";
-import displayMessage from "../code/nextTimeAllocate.js";
-import Timer from "../code/time.js";
-import "../code/expectedTime.js";
-
-const button = document.getElementById("startStopButton");
-const buttonClear = document.getElementById("clear");
-const text = document.getElementById("timer");
-const dropdown = document.getElementById("dropdown-contents");
-
-button.textContent = "Start";
-
-const timer = new Timer();
-
-button.addEventListener("click", () => {
-  if (button.textContent === "Start") {
-    if (dropdown.value == "") {
-      alert("You must pick a task before starting");
-      return;
-    }
-    const minutesSince = startTimer();
-    saveSessionState(dropdown);
-  } else {
-    stopTimer();
-  }
-  button.classList.toggle("stop");
-  text.classList.toggle("stopped");
-  button.textContent = button.textContent === "Start" ? "Stop" : "Start";
-});
-
-// Recover state
-
-loadActivities(() => {
-  if (isNotEmpty()) {
-    const [timeStarted, selectedIndex] = recoverSessionState();
-    const start = parseInt(timeStarted);
-
-    setTimeout(() => {
-      dropdown.selectedIndex = selectedIndex;
-      const now = Date.now();
-      const elapsed = Math.floor((now - start) / 1000);
-
-      timer.updateTimer(elapsed);
-      text.textContent = timer.getText();
-      button.click();
-    }, 1000);
-  }
-});
-
-// When the start button is clicked, the timer variable is initialized to 1 every second.
-const startTimer = () => {
-  if (timer.interval) return;
-  timer.interval = setInterval(() => {
-    timer.addOneSecond();
-    text.textContent = timer.getText();
-  }, 1000);
-};
-
-// Stops the increment of hours, minutes, and seconds on the timer.
-const stopTimer = () => {
-  clearInterval(timer.interval);
-  timer.interval = null;
-  cleanSessionState();
-
-  displayMessage(dropdown, timer.getText(), localStorage.getItem("expected"));
-  localStorage.removeItem("expected");
+import showExpectedTime from "../code/expectedTime.js";
+import showActivities from "../code/chooseActivities.js";
+import showMain from "../code/mainScreen.js";
+
+// Cool intro
+document.body.style.opacity = "1";
+
+const App = async () => {
```

Comment:
Now the app flow is a lot clearer, the app is just a series of screen, first we get the estimated time, then we get the activity
finally we show the main screen that it